recipes/guardian/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "guardian" %}
+{% set version = "1.3.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://git.ligo.org/cds/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5a6b76b842935e33cff0d123c6f71a994dfca3dc72994f9812afb12e218a8602
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - gpstime
+    - matplotlib
+    - networkx
+    - numpy
+    - pcaspy
+    - pydotplus
+    - pyepics
+    - python
+    - python-dateutil
+    - setproctitle
+    - termcolor
+
+test:
+  imports:
+    - guardctrl
+    - guardian
+  commands:
+    - guardian --help
+    - guardutil --help
+
+about:
+  home: https://git.ligo.org/cds/guardian/
+  dev_url: https://git.ligo.org/cds/guardian/
+  doc_url: https://docs.ligo.org/cds/guardian/
+  license: GPL-3.0-or-later
+  license_family: GPL
+  license_file: COPYING-GPL-3
+  summary: aLIGO Guardian
+  description: |
+    Design by and for the Advanced LIGO project, Guardian is a flexible,
+    modular, distributed, hierarchical, state machine automation platform.
+    It is not specific to aLIGO, though; it should be useful for an large
+    or small experiment that could benefit from state-machine supervisory
+    control.
+
+extra:
+  recipe-maintainers:
+    - duncanmmacleod
+    - jrollins
```

Comment:
@jrollins - are you OK with being a maintainer?

Reply:
Yes, absolutely.  Thank you @duncanmmacleod for building the recipe.